core/shared/src/main/scala/zio/Fiber.scala

```diff
@@ -403,13 +403,15 @@ object Fiber extends FiberPlatformSpecific {
     /**
      * Generates a fiber dump.
      */
-    final def dump: UIO[Fiber.Dump] =
+    final def dump(withTrace: Boolean): UIO[Fiber.Dump] =
```

Comment:
Do we expect generating a dump without tracing to be a common use case? We always capture traces today and don't seem to have had any issues with that.

If we do want this as an option I would suggest we change it to:

```scala
final def dumpWith(trace: Boolean): UIO[Fiber.Dump] = ???

val dump: UIO[Fiber.Dump] =
  dumpWith(true)
```

That way we have both the flexibility and the simple `dump` signature for the common case.

Reply:
I will add `dumpWith` to make this functionality accessible and and leave only one version in other places.

Reply:
@adamgfraser done!